ui/pages/home/home.component.js

```diff
@@ -300,20 +301,22 @@ export default class Home extends PureComponent {
                     infoText={error.data.snapId}
                     descriptionText={
                       <>
-                        <Typography
+                        <Text
+                          variant={TextVariant.bodyMd}
+                          as="h5"
                           color={TextColor.textAlternative}
-                          variant={TypographyVariant.H5}
                           fontWeight={FONT_WEIGHT.NORMAL}
                         >
                           {t('somethingWentWrong')}
-                        </Typography>
-                        <Typography
+                        </Text>
+                        <Text
                           color={TextColor.textAlternative}
-                          variant={TypographyVariant.H7}
+                          variant={TextVariant.bodySm}
+                          as="h7"
```

Comment:
There is no html element `h7` check mapping. Anything that is over `h6` should be `h6`
```suggestion
                          as="h6"
```